pkgs/games/dwarf-fortress/default.nix

```diff
@@ -24,30 +24,46 @@ let
   callPackage = pkgs.newScope self;
 
   df-games = lib.listToAttrs (map (dfVersion: {
-    name = "dwarf-fortress_${lib.replaceStrings ["."] ["_"] dfVersion}";
-    value = callPackage ./wrapper {
-      inherit (self) themes;
-      dwarf-fortress = callPackage ./game.nix { inherit dfVersion; };
-    };
+    name = "dwarf-fortress_${lib.replaceStrings ["."] ["_"] dfVersion}"; 
+    value =
+      let
+        # I can't believe this syntax works. Spikes of Nix code indeed...
+        dwarf-fortress = callPackage ./game.nix {
```

Comment:
I've inlined those JSON files. 

Maybe my knowledge of Nix is incomplete, but I think we might be stuck with the `let ... in` syntax here, because of the dependencies between packages?